### DISCUSSION TITLE
What are the expected charging models and costs to deploy, run and maintaining a program on Nillion?

### DISCUSSION THREAD

**Discussion by phuongvietvu0306** [NillionNetwork/discussions#43] [General]
Awesome project!
We are planning to build project depending a lot on Nillion and would like to know what are the expected charging models and costs to deploy, run and maintaining a program on Nillion when production is ready?

**Comment by oceans404** [comment]
Thank you!

Our team is working on defining the pricing models for storage and compute. 

What are you building? Would love to hear more about how we can support your use case.

**Reply by phuongvietvu0306** [reply]
Currently, our use cases are basic: Storing secrets, signing messages, and verifying hashes
It would be appreciated if you could disclose some aspects considered when determining pricing models and when these models may be made publicly available

We are still looking for a solution and Nillion is top of the list. We look forward to hearing information soon to complete our financial plan and begin product development
